spyder/plugins/switcher/plugin.py

```diff
@@ -89,3 +146,85 @@ def on_main_menu_teardown(self):
 
     # --- Public API
     # ------------------------------------------------------------------------
+
+    # Switcher methods
+    def set_placeholder_text(self, text):
+        """Set the text appearing on the empty line edit."""
+        self._switcher.set_placeholder_text(text)
+
+    def setup(self):
+        """Set-up list widget content based on the filtering."""
+        self._switcher.setup()
+
+    def set_position(self, top, parent=None):
+        """Set the position of the dialog."""
+        self._switcher.set_position(top, parent)
+
```

Comment:
```suggestion
```

This method doesn't seem to be necessary because it's not used in `EditorSwitcherManager`, only internally in the Switcher container.